images/os/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fedora:29 AS build
+
+COPY --from=registry.svc.ci.openshift.org/openshift/origin-v4.0:machine-os-content /srv/ /srv/
+RUN yum install -y ostree yum-utils selinux-policy-targeted && \
+    commit=$( find /srv -name *.commit | sed -Ee 's|.*objects/(.+)/(.+)\.commit|\1\2|' | head -1 ) && \
+    mkdir /tmp/working && cd /tmp/working && \
+    yumdownloader --destdir=/tmp/rpms origin-node origin-clients && \
+    for i in $(find /tmp/rpms/ -name origin-*); do rpm2cpio $i | cpio -div; done && \
+    ostree --repo=/srv/repo commit --parent=$commit --tree=ref=$commit --tree=dir=. --selinux-policy / \
+        -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev
```

Comment:
So, `--selinux-policy /` here is using the file contexts from the f29 buildroot, but we're running in RHEL. Probably fine if none of the files need special labels, but why not use a CentOS buildroot with CentOS' `selinux-policy-targeted`?

Reply:
Hmm, yeah. So was looking at `ostree ls -RX origin-ci-dev` and it's basically all `etc_t` and `bin_t` (we're also including manpages right now, but meh... don't need to overoptimize here). Anyway, seems safe enough!